src/css/imports/captions.less

```diff
@@ -13,6 +13,10 @@
         &.jw-captions-enabled {
             display: block;
         }
+
+        &.jw-captions-disabled {
+            display: none;
```

Comment:
This is for explicitly disabling subtitles in relation to casting.   May be able to revisit and remove this depending on the cast provider work